tests/test_calculate_metrics.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import pandas as pd
+from numpy.testing import assert_array_equal
```

Comment:
This makes sense. Also, I didn't know about the numpy.testing library, seems like a useful tool for other tests we might want to write. Thanks!

Reply:
There is also one provided by pandas (for series and dataframe comparison)